### ISSUE TITLE
Species present in ITIS not showing up on backbone taxonomy

### ISSUE THREAD

**Issue by mikeroswell** [gbif/portal-feedback#3064]
Not sure how this occurs. *Xylocopa senior* appears to be a [valid species name](https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=766882#null), but doesn't show up in the GBIF backbone taxonomy. I wouldn't expect this behavior based on my current understanding of the backbone taxonomy, suspect a bug.

**Comment by ManonGros** [comment]
As far as I can see, this species isn't included in the version of [ITIS currently published on GBIF](https://www.gbif.org/dataset/9ca92552-f23a-41a8-a140-01abaa31c931) nor in any other checklist available on GBIF: https://www.gbif.org/species/search?q=Xylocopa%20senior

The data has to be shared on GBIF before being integrated in the backbone (more information on [this blogpost](https://data-blog.gbif.org/post/gbif-backbone-taxonomy/)).

Now I am not sure why *Xylocopa senior* isn't in the ITIS checklist we have. Any idea @mdoering?

**Comment by mdoering** [comment]
I cannot see in ITIS when the record was added so it's hard to tell.
@thomasstjerne 

The GBIF dataset was built on the September 28, 2020 version of the [ITIS mysql dump](https://www.itis.gov/downloads/index.html).

[ITIS in COL](https://data.catalogue.life/dataset/2144/meta) uses a new ColDP based format and is from 28th October, their latest dump. There you can find [Xylocopa senior](https://data.catalogue.life/dataset/2144/taxon/766882) which will make its way into the next COL release which we'll in turn use for the upcoming backbone.

**Comment by mdoering** [comment]
The latest unreleased version of COL already has it: https://data.catalogue.life/dataset/3/taxon/486a9847-7152-4186-83f3-66b0d1f47c0e

**Comment by mikeroswell** [comment]
So is the upshot that there was something weird going on where for some reason this species was not in ITIS for reasons probably not related to any kind of "bug" in any of the above referenced DBs? Thanks for looking into this!

**Comment by mdoering** [comment]
the main thing surely is that the current GBIF backbone is from end of 2019

**Comment by ManonGros** [comment]
@mdoering although _Xylocopa senior_ has been in the CoL since November, it still isn't in the backbone for some reason. Could you take a look and see if this has to do with the backbone building?

**Comment by ahahn-gbif** [comment]
After the latest backbone refresh, _X. senior_ does show in the CoL checklist, based on ITIS (https://www.gbif.org/species/search?q=Xylocopa%20senior&dataset_key=7ddf754f-d193-4cc9-b351-99906754a03b&origin=SOURCE&advanced=1), but is not part of the backbone taxonomy. Could this be due to resource ranking in backbone processing?

**Comment by ahahn-gbif** [comment]
@mdoering: I don't think we can do anything about this from here. I am assigning it to you, but feel free to ping us if there is anything needed in terms of communication
